xyz

```diff
@@ -90,29 +138,25 @@ while (($# > 0)) ; do
       exit
       ;;
     -v|--version)
-      node -p "require('$dir/package.json').version"
+      node -p 'require("xyz/package.json").version'
```

Comment:
This change means `./xyz --version` no longer works (from this project's root directory), but I don't consider this a problem. Not having to determine our location on the file system is a win.